configuration.json

```diff
@@ -0,0 +1,9 @@
+{
+	"path_data": "/path/to/data/",
+	"subjects": "sub-101, sub-102",
+	"data_type":"anat",
+	"contrast":"t1",
```

Comment:
this is a bit confusing-- this JSON is dealing with both preprocessing and template generation-- consider splitting